WordPress/Classes/ViewRelated/Blog/Blog List/BlogListViewController.m

```diff
@@ -525,6 +531,19 @@ - (void)instantiateNoResultsViewControllerIfNeeded
 }
 
 #pragma mark - Notifications
+- (void)registerForPostSignUpNotifications
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(launchSiteCreation)
+                                                 name:@"PSICreateSite"
+                                               object:nil];
+
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(showLoginControllerForAddingSelfHostedSite)
+                                                 name:@"PSIAddSelfHosted"
```

Comment:
Wiring the notifications, as strings, is very very very fragile. Suggestion: just declare the [Notifications as strings, directly](https://stackoverflow.com/questions/46818582/how-to-use-notification-name-extension-from-swift-3-to-objective-c)